pkg/api/types.go

```diff
@@ -188,6 +188,8 @@ const (
 	NamespaceNone string = ""
 	// NamespaceSystem is the system namespace where we place system components.
 	NamespaceSystem string = "kube-system"
+	// NamespacePublic is the namespace where we place public info (ConfigMaps)
+	NamespacePublic string = "kube-public"
```

Comment:
I've added auto-create without a flag.  This is low impact enough I think it should be cool.